src/serial.rs

```diff
@@ -96,11 +97,32 @@ pub mod config {
         STOP1P5,
     }
 
+    pub enum BitOrder {
+        Lsb,
+        Msb
+    }
+
+    pub enum ClockPhase {
+        First,
+        Second
+    }
+
+    pub enum ClockPolarity {
+        High,
+        Low
+    }
+
     pub struct Config {
         pub baudrate: Hertz,
         pub wordlength: WordLength,
         pub parity: Parity,
         pub stopbits: StopBits,
+        pub clock_phase: ClockPhase,
```

Comment:
It looks like the naming convention in this struct is without the underscore

Reply:
<img width="323" alt="image" src="https://user-images.githubusercontent.com/7644264/132079864-d53a4dfe-6e37-4fa9-b5b7-8f1c33a38983.png">
so something like this?

Reply:
👍